test/jdk/javax/swing/JMenu/4213634/bug4213634.java

```diff
@@ -39,85 +39,79 @@
  * @test
  * @key headful
  * @bug 4213634 8017187
- * @author Scott Violet
  * @library ../../regtesthelpers
  * @build Util
+ * @summary Verifies if Alt+mnemonic char works when
+ *          menu & menuitem have same mnemonic char
  * @run main bug4213634
  */
 
-
 public class bug4213634 {
 
-    private JMenu menu;
-
-    private JFrame frame;
-
-    public static void main(String[] args) throws Throwable {
-        new bug4213634();
-    }
+    private static JMenu menu;
+    private static JFrame frame;
+    private static Robot robot;
 
-    bug4213634() throws AWTException, InterruptedException, InvocationTargetException {
-        SwingUtilities.invokeAndWait(new Runnable() {
-            @Override
-            public void run() {
-                createAndShowGUI();
-            }
-        });
+    public static void main(String[] args) throws Exception {
+        try {
+            robot = new Robot();
+            SwingUtilities.invokeAndWait(() -> createAndShowGUI());
 
-        test();
+            robot.waitForIdle();
+            robot.delay(1000);
+            test();
+        } finally {
+            SwingUtilities.invokeAndWait(() -> {
+                if (frame != null) {
+                    frame.dispose();
+                }
+            });
+        }
     }
 
-    public  void createAndShowGUI() {
-        frame = new JFrame("TEST");
+    public static void createAndShowGUI() {
+        frame = new JFrame("bug4213634");
         JMenuBar mb = new JMenuBar();
         menu = mb.add(createMenu("1 - First Menu", true));
         mb.add(createMenu("2 - Second Menu", false));
         frame.setJMenuBar(mb);
-        JTextArea ta = new JTextArea("This test dedicated to Nancy and Kathleen, testers and bowlers extraordinaire\n\n\nNo exception means pass.");
-        frame.getContentPane().add("Center", ta);
         JButton button = new JButton("Test");
         frame.getContentPane().add("South", button);
-        frame.setBounds(100, 100, 400, 400);
+        frame.setSize(300, 300);
+        frame.setLocationRelativeTo(null);
         frame.setVisible(true);
         button.requestFocusInWindow();
     }
 
-    private void test() throws AWTException, InterruptedException, InvocationTargetException {
-        Robot robot = new Robot();
-        robot.setAutoDelay(50);
-        robot.waitForIdle();
+    private static void test() throws Exception {
 
         Util.hitMnemonics(robot, KeyEvent.VK_1);
```

Comment:
```suggestion
    private static void test() throws Exception {
        Util.hitMnemonics(robot, KeyEvent.VK_1);
```
The blank line at the start of a method is redundant and may be removed safely.